src/main/java/hudson/remoting/MimicException.java

```diff
@@ -29,7 +33,8 @@ public String toString() {
         return (message != null) ? (s + ": " + message) : s;
     }
 
-    public static Throwable make(Channel ch, Throwable cause) {
+    @Nullable
```

Comment:
@stephenc This is how we handle such use-cases in other places (IIRC I was also requested to do so at some point). Here are two examples in the core:

* https://github.com/jenkinsci/jenkins/blob/461472fc009dfe313009acd8ce23cff22ee66b91/core/src/main/java/jenkins/SoloFilePathFilter.java#L24
* https://github.com/jenkinsci/jenkins/blob/c3b1bff6477e05050a40c0e20816bea9cf396534/core/src/main/java/jenkins/model/Jenkins.java#L1997

Although I agree it would better to have a `CheckForNull` (or even `Nonnull`) method, we need to agree on the approach